scanomatic/ui_server_data/js/som/analysis.js

```diff
@@ -20,62 +46,52 @@ function set_fixture_plate_listing() {
       } else {
         $('#manual-regridding-settings').hide();
       }
-      parent = $('#manual-regridding-plates');
+      const parent = $('#manual-regridding-plates');
       parent.empty();
-      Map(gridplates, (e) => { append_regridding_ui(parent, e); });
+      Map(gridplates, (e) => { appendRegriddingUI(parent, e); });
     }
   };
 
-  error_callback = function () {
+  const errorCallback = () => {
     $('#fixture-error-message').html('<em>Fixture file missing</em>').show();
   };
 
   if (localFixture) {
     if (path.length > 5) {
-      $.get(`/api/data/fixture/local/${path.substring(5, path.length)}`, callback).fail(error_callback);
+      $.get(`/api/data/fixture/local/${path.substring(5, path.length)}`, callback).fail(errorCallback);
     } else {
-      error_callback();
+      errorCallback();
     }
   } else {
-    fixt = $(current_fixture_id).val();
+    const fixt = $(currentFixtureId).val();
```

Comment:
I think this might be a bit of a confusion. I saw that this was defined in the global scope in the html-code in various places but now it's defined here in the module... it should probably be either.

Reply:
There were some cases of this in various html-files

Reply:
yes, probably same case as `selectedFixtureCanvasJq` above... Well spotted.

Reply:
I think I've fixed all of them.